test/external/process_replay/process_replay.py

```diff
@@ -11,16 +11,19 @@
 for offset in tqdm(range(0, row_count, page_size)):
   cur.execute(f"SELECT val FROM 'process_replay_{VERSION}' LIMIT ? OFFSET ?", (page_size, offset))
   for row in cur.fetchall():
-    compare_k, compare_src = pickle.loads(row[0])
+    compare_k, compare_src, time_baseline = pickle.loads(row[0])
     k = Linearizer(*compare_k.ast, opts=compare_k.opts)
     for opt in compare_k.applied_opts: k.apply_opt(opt)
     good_src = k.opts.render(to_function_name(compare_k.name), k.linearize().uops)
-    try: assert compare_src == good_src
+    time_ = time_lin(k)
+    try:
+      assert compare_src == good_src, "PROCESS REPLAY DETECTED CHANGE"
+      assert (time_-time_baseline)/max(time_baseline, .1) < 0.1, f"TIME REPLAY DETECTED CHANGE {time_baseline} ->{time_}"
```

Comment:
Let's remove this assert for now and just print - We need to stress test flakiness before making it a merge blocker.